src/operators/rastertypeconversion.md

```diff
@@ -0,0 +1,42 @@
+# RasterTypeConversion
+
+The `RasterTypeConversion` operator allows change the data type of raster data.
```

Comment:
```suggestion
The `RasterTypeConversion` operator allows changing the data type of raster data.
```

I recommend installing the Grammarly plugin for VSCode.

Reply:
[f61834f](https://github.com/geo-engine/docs/pull/29/commits/f61834f48103b630a6d859ffcffe669c977bceb0)

Reply:
the plugin does not work :(